src/main/java/com/stripe/net/StripeResponseGetter.java

```diff
@@ -18,6 +18,19 @@ <T extends StripeObjectInterface> T request(
       ApiMode apiMode)
       throws StripeException;
 
+  @SuppressWarnings("TypeParameterUnusedInFormals")
+  default <T extends StripeObjectInterface> T request(ApiRequest request, Type typeToken)
+      throws StripeException {
+    return request(
```

Comment:
Should we deprecate the old request method?

Reply:
SGTM

Reply:
Done in [Address codereview (deprecate, fullurl)](https://github.com/stripe/stripe-java/pull/1702/commits/a2262ab34877d7159e52618c98db992385815fff)